README.md

```diff
@@ -0,0 +1,179 @@
+
+## Viam configuration
+
+### Configure module
+
+This module should first be configured as follows:
+
+In the Viam app, go to Config->Modules and add a new module.
+Choose any `Name` you'd like, and for `Executable path` add the path to the executable you generated in the [Build step](#build) (for example, */home/username/periph_board/periph_board*).
+
+Make sure to `Save config` at the bottom of the screen.
+
+If you prefer to edit raw configuration JSON, make sure inside the main block your `modules` config looks something like this:
+
+``` json
+{
+  "modules": [
+    {
+      "executable_path": "/home/username/periph_board/periph_board",
+      "name": "periph"
+    }
+  ],
+  "components": []
+}
+```
+
+### Configure component
+
+After configuring the module, configure the component as type *board*, model *viamlabs:board:periph*.
+You may choose any `Name` you want.
+
+If you prefer to edit raw configuration JSON, make sure inside the main block your `components` config looks something like this:
+
+``` json
+{
+  "modules": [
+    {
+      "executable_path": "/home/username/periph_board/periph_board",
+      "name": "periph"
+    }
+  ],
+  "components": [
+    {
+      "name": "local",
+      "type": "board",
+      "model": "viamlabs:board:periph",
+      "attributes": {},
+      "depends_on": []
+    }
+  ]
+}
+```
+
+## viam-server board setup examples
+
+### Orange Pi Zero 2
+
+The [Orange Pi Zero 2](http://www.orangepi.org/html/hardWare/computerAndMicrocontrollers/details/Orange-Pi-Zero-2.html) is an open-source SBC with dual-band wireless WiFi and Bluetooth 5.0 .It is highly compact with a dimension of 53X60mm. It can run Android10, Ubuntu, Debian.
+
+Start by downloading the latest Linux distribution from OrangePi.
+We tested and had success with the [Debian bullseye distribution](https://drive.google.com/drive/folders/1Xk7b1jOMg-rftowFLExynLg0CyuQ7kCM).
+
+Flash this image to an SD card with [Balena Etcher](https://etcher.balena.io/)
+
+Then insert the flashed SD card to the Orange Pi, plug the Pi into a monitor with an mini-HDMI cable, and plug in the Pi to with a 5v power supply.
+Log in with the credentials root/orangepi.
+
+Secure the Pi:
+
+- Change the root password by issuing the `passwd` command.
+- Change the password for the `orangepi` user account by issuing the commmand `passwd orangepi`
+
+Update the software:
+
+``` bash
+apt update
+apt upgrade
+```
+
+Enable wifi:
+
+```bash
+nmcli dev wifi
+nmcli dev wifi connect <wifi_name> password <wifi_passwd>
+```
+
+Enable the host being announced on the network by running `orangepi-config` and going to System->Avahi and *Enable system announce in the network*.
+
+While still in `orangepi-config`, change the hostname to your preference by going to Personal->Hostname.
+
+Enable i2c and spi by editing `/boot/orangepiEnv.txt`, adding the line:
+
+``` bash
+overlays=i2c3 spi-spidev
+param_spidev_spi_bus=1
+param_spidev_spi_cs=1
+```
+
+Now install the Orange Pi gpio tool:
```

Comment:
The above overlays are actually needed for Viam to function properly, but the below WiringOP stuff should be fully unrelated to Viam and is just a coincident debug tool; not sure it needs to be part of the setup.

Reply:
I found the online docs for this board to be hard to use to find pin numbers, but this tool makes it easy.  I added a comment as to why in the doc